Gemfile

```diff
@@ -47,8 +47,8 @@ group :backend do
   gem 'webdrivers', require: false
 
   # JavaScript testing
-  gem 'teaspoon', github: 'jejacks0n/teaspoon', require: false
```

Comment:
what about moving it in the gemspec at this point?

Reply:
We're not tracking dev dependencies in gemspecs, so it should stay there for consistency